src/Core/Geographic/Extent.js

```diff
@@ -90,7 +92,37 @@ Extent.prototype.as = function as(crs) {
     assertCrsIsValid(crs);
 
     if (_isTiledCRS(this._crs)) {
-        throw new Error('Unsupported yet');
+        if (crs === 'EPSG:3857') {
+            const nbCol = Math.pow(2, this.zoom);
+            const size = 360 / nbCol;
+            const west = 180 - size * (nbCol - this.col);
+            const east = 180 - size * (nbCol - (this.col + 1));
+            const nbRow = nbCol;
+            const sizeRow = 1.0 / nbRow;
+            const Yn = 1 - sizeRow * (nbRow - (this.row));
+            const Ys = 1 - sizeRow * (nbRow - (this.row + 1));
+            const north = THREE.Math.radToDeg(projection.YToWGS84(Yn));
+            const south = THREE.Math.radToDeg(projection.YToWGS84(Ys));
+            const minCoord = new Coordinates('EPSG:4326', west, south).as(crs);
+            const maxCoord = new Coordinates('EPSG:4326', east, north).as(crs);
+            return new Extent(crs, {
```

Comment:
Maybe add a comment explaining what you're doing because it's not obvious.

Reply:
Also it seems weird to reference 4326 here and have a completely different computation 5 lines below for 4326?

Reply:
i add comment
and fix error state crs